cmd/minishift/cmd/stop.go

```diff
@@ -61,12 +61,16 @@ func runStop(cmd *cobra.Command, args []string) {
 
 	fmt.Println("Stopping local OpenShift cluster...")
 
-	if !skipUnRegistration {
-		if _, err := cluster.UnRegister(api); err != nil {
+	if !skipUnRegistration && minishiftConfig.InstanceConfig.IsRegistered {
+		supportUnregistration, err := cluster.UnRegister(api)
```

Comment:
>If minishiftConfig.InstanceConfig.IsRegistered was the whole point, it seems we neglected to actually check the state of this variable.

@gbraad right it was missed from my sight when implementing `minishiftConfig.InstanceConfig.IsRegistered` flag and should be handled same as delete command which only look for that variable.